algebra/tests/quat/various.c

```diff
@@ -1053,3 +1066,90 @@ TEST_GROUP_RUNNER(group_quat_vecRot)
 	RUN_TEST_CASE(group_quat_vecRot, quat_vecRot_biggerValues);
 	RUN_TEST_CASE(group_quat_vecRot, quat_vecRot_zeroVector);
 }
+
+
+/* ##############################################################################
+ * ------------------        quat_axisAngle2quat tests       --------------------
+ * ############################################################################## */
+
+
+TEST_GROUP(group_quat_axisAngle2quat);
+
+
+TEST_SETUP(group_quat_axisAngle2quat)
+{
+}
+
+
+TEST_TEAR_DOWN(group_quat_axisAngle2quat)
+{
+}
+
+
+TEST(group_quat_axisAngle2quat, quat_axisAngle2quat_baseQuaternions)
+{
+	quat_t q, qA, qI, qJ, qK, qZero = { 0 };
+	vec_t v = { .x = 1.0f, .y = 0.0f, .z = 0.0f };
```

Comment:
This may imporve readibility and avoid _peaky_ setting ones and zeroes later in the test.

```suggestion
	const vec_t versX = { .x = 1.0f, .y = 0.0f, .z = 0.0f };
	const vec_t versY = { .x = 0.0f, .y = 1.0f, .z = 0.0f };
	const vec_t versZ = { .x = 0.0f, .y = 0.0f, .z = 1.0f };
	

```